index.php

```diff
@@ -1102,8 +1102,6 @@
 								<span id="helioviewer-url-box-msg"></span>
 								<form style="margin-top: 5px; text-align: center;">
 									<input aria-label="Viewport URL" type="text" class="helioviewer-url-input-box helioviewer-url-input-box-menu" style="width:98%;" value="https://helioviewer.org" />
-									<label for="helioviewer-url-shorten-1">Shorten with bit.ly? </label>
-									<input id="helioviewer-url-shorten-1" type="checkbox" class="helioviewer-url-shorten" />
```

Comment:
Actually, can you add this back and just change the text from "Shorten with bit.ly" to just "Shorten link"

Reply:
Hey, I have created new PR here: https://github.com/Helioviewer-Project/helioviewer.org/pull/471